settings.gradle.kts

```diff
@@ -14,10 +14,13 @@ plugins {
   id("com.android.settings") version "8.8.0" // Keep in sync with agp in libs.versions.toml
 }
 
+val isCI = System.getenv("CI") != null
+
 develocity {
   buildScan {
     termsOfUseUrl.set("https://gradle.com/help/legal-terms-of-use")
     termsOfUseAgree.set("yes")
+    uploadInBackground = !isCI
```

Comment:
This also makes sense to me. Just checking you wanted it in this PR

Reply:
yes, just snuck it in here to have one less PR to check